less/2016/na/styles.less

```diff
@@ -0,0 +1,5719 @@
+ text-indent:0;
+ border:none;
+ box-shadow:none;
+ line-height:normal;
+ list-style:none;
+ padding:1em 0.85em;
+ font-weight:normal
+}
+#main-menu ul.nav-bar li a.flyout-toggle {
+ padding:0.65em 1.1em
+}
+#main-menu ul.nav-bar li a {
+ font-weight:normal;
+ font-size:2em;
+ color:#fff;
+ border:none;
+ padding:0 .9em 0 0;
+ line-height:1.2
+}
+#main-menu ul.nav-bar li a:hover,
+#main-menu ul.nav-bar li a:focus {
+ color:#fff;
+ text-decoration:underline
+}
+#main-menu ul.nav-bar li a.active {
+ font-weight:normal;
+ color:#FFF;
+ text-decoration:none
+}
+#main-menu ul.nav-bar li a.active :hover,
+#main-menu ul.nav-bar li a.active :focus {
+ color:#fff;
+ text-decoration:none
+}
+#main-menu ul.nav-bar li:hover,
+#main-menu ul.nav-bar li:focus {
+ background:none;
+ color:#FFF;
+ text-decoration:none
+}
+#main-menu ul.nav-bar li ul.flyout {
+ background:none repeat scroll 0 0 #ab3155;
+ border-color:#ab3155;
+ position:static;
+ text-indent:0;
+ top:4.275em;
+ margin-left:1px;
+ visibility:inherit
+}
+#main-menu ul.nav-bar li ul.flyout li {
+ font-size:0.8em;
+ font-weight:normal
+}
+#main-menu ul.nav-bar li ul.flyout li a {
+ font-weight:normal;
+ background:none !important;
+ border:hidden !important;
+ box-shadow:none
+}
+#main-menu ul.nav-bar li:hover {
+ background:none repeat scroll 0 0 #ab3155;
+ border-color:#ab3155
+}
+#main-menu ul.nav-bar li.has-flyout a:first-child:after {
+ right:0.5em;
+ top:0.5em
+}
+#main-menu #call-for-action-area a {
+ text-align:center;
+ display:block;
+ margin:10px auto 0;
+ max-width:184px
+}
+#secondary-menu {
+ float:left;
+ padding:12px 0 12px 30px
+}
+#secondary-menu li {
+ font-family:Arial,Helvetica,"Nimbus Sans L",sans-serif;
+ font-weight:bold;
+ font-size:0.857em;
+ margin:0 0.833em
+}
+#secondary-menu li a {
+ color:#b7b7b7
+}
+#secondary-menu li a.active {
+ color:#FFF
+}
+#secondary-menu li a:hover,
+#secondary-menu li a:focus {
+ color:#FFF;
+ text-decoration:underline
+}
+#secondary-menu li.first {
+ margin-left:0 !important
+}
+#secondary-menu li.last {
+ margin-right:0 !important
+}
+#navigation section#main-menu .three {
+ padding:0
+}
+.region-social {
+ text-align:right
+}
+.region-social ul.menu li {
+ padding:0 5px
+}
+.region-social ul.menu li a {
+ color:#fff;
+ font-size:2em
+}
+.region-social ul.menu li a:hover {
+ opacity:.75
+}
+@media all and (max-width: 767px) {
+ #navigation #main-menu .row nav.columns li.selected-element {
+  background-color:#f7d4a2
+ }
+ #navigation #main-menu .row nav.columns li.selected-element a {
+  text-shadow:none !important
+ }
+ #navigation #main-menu .row nav.columns li.selected-element a.active {
+  color:#fff;
+  text-shadow:none
+ }
+}
+@media all and (max-width: 980px) {
+ #twitter-area {
+  display:none
+ }
+ #navigation #main-menu .row nav.columns {
+  width:99%;
+  margin:0 auto
+ }
+}
+@media all and (min-width: 768px) {
+ body.admin-menu {
+  margin-top:2em !important
+ }
+ #page {
+  position:relative;
+  padding-top:6.3em
```

Comment:
Can you remove the padding here. It adds a lot of extra un-needed space.

Reply:
Updated.